app/javascript/packages/document-capture/context/acuant.tsx

```diff
@@ -268,6 +269,15 @@ function AcuantContextProvider({
         loadAcuantSdk();
       }
       window.AcuantJavascriptWebSdk = getActualAcuantJavascriptWebSdk();
+
+      window.AcuantJavascriptWebSdk.setUnexpectedErrorCallback((error_message) => {
+        trackEvent('idv_sdk_error_before_init', {
+          success: false,
+          error_message,
```

Comment:
I'm not sure what to do about this yarn lint error:

`error  Identifier 'error_message' is not in camel case  camelcase`

If I change it to camelcase, then on the `analytics_events.rb` side it will have the opposite complaint - that it should be snake case.

Reply:
I _think_ the linter might be okay if you pass the key as snake case, but reference the value as camelCase.

```suggestion
          error_message: errorMessage,
```

Reply:
That fixed it, thank you! 🎉

Reply:
Thank you for catching that! Fixed in 535a9305981c43a45cead51d09d89693cb90deed.